includes/modules/payment/securesubmit.php

```diff
@@ -219,7 +221,19 @@ public function before_process() {
 
         return false;
     }
-
+    
+    protected function getConfig()
+    {
+        $config = new ServicesConfig();
+        $config->secretApiKey = MODULE_PAYMENT_SECURESUBMIT_SECRET_API_KEY;
+        $config->serviceUrl = ($this->enableCryptoUrl) ?
```

Comment:
We can scrap the `enableCryptoUrl` property and only use `https://api2.heartlandportico.com` and `https://cert.api2.heartlandportico.com`.

Reply:
updated